controller/Honeypot.php

```diff
@@ -85,7 +85,7 @@ public function getEncryptedTime()
     public function decryptTime($time)
     {
         try {
-            return base64_decode($time);
+            return intval(base64_decode($time));
```

Comment:
Should this change be reflected at line 5?

Reply:
The explanation on line 5 is a bit vague perhaps, but I don't think it needs to be changed due to this. It's not more wrong than it used to be ;)